js/main.js

```diff
@@ -23,17 +23,11 @@
     window.pin.mapNode.classList.remove(`map--faded`);
     window.form.formNode.classList.remove(`ad-form--disabled`);
     toggleDisabledOnFormNodes();
+    window.form.passAddressInput(window.move.MainPinSize.pin.WIDTH, window.move.MainPinSize.pin.HEIGHT);
   };
 
-  toggleDisabledOnFormNodes();
-
-  window.map.mapPinMain.addEventListener(`click`, function () {
-    onActiveMode();
-    window.map.initPinsScreen();
-    window.form.passAddressInput();
-
+  const onPinsClick = () => {
     let pinsArr = Array.from(window.pin.mapPinsNode.querySelectorAll(`.map__pin:not(.map__pin--main)`));
```

Comment:
Тут кстати не обязательно приводить к массиву
```suggestion
    let pinsArr = window.pin.mapPinsNode.querySelectorAll(`.map__pin:not(.map__pin--main)`);
```